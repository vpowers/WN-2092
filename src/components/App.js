import React from 'react';

const App = () => (
  <div>
    <h1>React+Redux Boilerplate</h1>
  </div>
);

export default App;