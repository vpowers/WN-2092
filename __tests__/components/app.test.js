import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import App from '../../src/components/App';

describe('<App />', () => {
    let component, editButton, deleteButton, props;

    it('it should render App', () => {
        component = shallow(<App />);
        expect(toJson(component)).toMatchSnapshot();
    });
});