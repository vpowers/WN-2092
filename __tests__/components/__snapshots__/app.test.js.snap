// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> it should render App 1`] = `
<div>
  <h1>
    React+Redux Boilerplate
  </h1>
</div>
`;
